wcm-smoke/src/main/java/com/adobe/cq/cloud/testing/it/wcm/smoke/DeepGetPageIT.java

```diff
@@ -146,11 +169,7 @@ private static void verifyPageAndResources(HtmlUnitClient client, String path) t
     private static boolean isSameOrigin(URI uri1, URI uri2) {
         if (!uri1.getScheme().equals(uri2.getScheme())) {
             return false;
-        } else if (!uri1.getAuthority().equals(uri2.getAuthority())) {
-            return false;
-        } else {
-            return true;
-        }
+        } else return uri1.getAuthority().equals(uri2.getAuthority());
```

Comment:
maybe add the return into {} as the code at other places uses braces everywhere for single line statements

Reply:
or rather remove the else keyword and add a linebreak